plugin/depsusage/packagehint.go

```diff
@@ -30,3 +33,67 @@ func resolvePythonPackageHint(s string) string {
 	}
 	return s
 }
+
+func resolveGoPackageHint(moduleName string) string {
```

Comment:
Now that I see `resolveGoPackageHint`, I think it has a lot of business logic and failure cases. We should not silently fail by returning `""`. Instead we should probably update the method signature to `(string, error)` to explicitly fail and let the caller handle the error. This is a breaking change but I am guessing on `vet` is impacted. Since we are not yet stable, I think its OK to have a breaking change.